### PR TITLE
Separate accessors

### PR DESCRIPTION
**Pull request by mbstrange2**
Separate controller paths for each input and output of the memory tile...

### CODE REVIEW

File: lake/utils/test_infra.py [hunk 1]
```diff
@@ -87,15 +87,17 @@ def gen_test_lake(config_path,
         tester.step(2)
 
     with tempfile.TemporaryDirectory() as tempdir:
+        tempdir = "temp_separate"
```
Comment (kavyasreedhar) [inline lake/utils/test_infra.py:90]: delete

File: lake/utils/test_infra.py [hunk 2]
```diff
@@ -87,15 +87,17 @@ def gen_test_lake(config_path,
...
         tester.compile_and_run(target="verilator",
                                directory=tempdir,
-                               flags=["-Wno-fatal"])
+                               flags=["-Wno-fatal", "--trace"])
```
Comment (kavyasreedhar) [inline lake/utils/test_infra.py:93]: delete trace

File: tests/test_lake.py [hunk 3]
```diff
@@ -3,7 +3,7 @@
 
 
 # add more tests with this function by adding args
-@pytest.mark.parametrize("args", [lake_test_app_args("conv_3_3")])
+@pytest.mark.parametrize("args", [lake_test_app_args("separate")])
```
Comment (kavyasreedhar) [inline tests/test_lake.py:6]: Can separate be added as a second value for args so we keep the first test too?

**Review by kavyasreedhar — CHANGES_REQUESTED** [review]

**Review by kavyasreedhar — APPROVED** [review]

**Comment by mbstrange2** [comment]
Passes AHA PR https://github.com/StanfordAHA/aha/pull/1025

https://buildkite.com/stanford-aha/aha-flow/builds/4066#288af191-a248-494f-b13f-403adb32693f

**Pull request merged**